src/org/pursuit/Main.java

```diff
@@ -212,19 +196,27 @@ private void viewHomePage() {
 
     }
 
-    private Integer getAccountByCustomerName() {
-        System.out.println("What is the account number to the account you would like to view");
-        int userAccountNumberInput = input.nextInt();
+    //this needs to be renamed
+    private void displayAccountByAccountNumber(int accountNumber) {
 
```

Comment:
Here, you could have just called   System.out.println(accountMap.get(integer)); in a one liner, without using any for loop. If you really want null safety check here, you could first check if accountMap.containsKey(integer)